readme.md

```diff
@@ -93,6 +93,36 @@ prompt pure
 | **`PURE_GIT_DOWN_ARROW`**        | Defines the git down arrow symbol.                                                             | `⇣`            |
 | **`PURE_GIT_UP_ARROW`**          | Defines the git up arrow symbol.                                                               | `⇡`            |
 
+## Colors
+
+Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fzutil-Module) with a pattern of the form `:prompt:pure:$color_name` and style `color`. The color names, their default and what part they affect are:
+- `branch` (242), the name of the current branch when in a git repository
+- `branch:cached` (red), the name of the current branch when the data isn't fresh
+- `exec_time` (yellow), the execution time of the last command when exceeding `PURE_CMD_MAX_EXEC_TIME`
+- `git_arrow` (cyan), for `PURE_GIT_UP_ARROW` and `PURE_GIT_DOWN_ARROW`
```

Comment:
I see you switched to categories/subcategories, I think it's a good idea. Should we do it for `git` as well? E.g. `git:arrow`.